vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -500,7 +500,7 @@ - (void)reportLoadedPageWithURL:(NSURL *)url redirectedFromURLs:(NSArray<NSURL *
     urls.push_back(base::SysNSStringToUTF8(redirectURL.absoluteString));
   }
   urls.push_back(urlString);
-  ads->OnPageLoaded((int32_t)tabId, urls, base::SysNSStringToUTF8(text));
+  ads->OnPageLoaded((int32_t)tabId, -1, false, urls, base::SysNSStringToUTF8(text));
```

Comment:
Passing -1 for page_transition and false for has_user_gesture so that iOS does not use the improved user activity event types as they are not required at this time